classes/LocalizationPack.php

```diff
@@ -36,11 +36,24 @@ class LocalizationPackCore
     protected $iso_currency;
     protected $_errors = array();
 
-    public function loadLocalisationPack($file, $selection, $install_mode = false, $iso_localization_pack = null)
+    /**
+     * Loads localization pack
+     *
+     * @param SimpleXMLElement|string $pack                  Localization pack as SimpleXMLElement or plain XML string
+     * @param array                   $selection             Content to import selection
+     * @param bool                    $install_mode          Whether mode is installation or not
+     * @param string|null             $iso_localization_pack Country Alpha-2 ISO code
+     *
+     * @return bool
+     */
+    public function loadLocalisationPack($pack, $selection, $install_mode = false, $iso_localization_pack = null)
     {
-        if (!$xml = @simplexml_load_string($file)) {
+        if ($pack instanceof SimpleXMLElement) {
+            $xml = $pack;
+        } elseif (!$xml = @simplexml_load_string($pack)) {
```

Comment:
There is a `libxml_clear_errors`, instead using `@` you can use `libxml_use_internal_errors(true)` : http://php.net/manual/fr/function.libxml-use-internal-errors.php

Reply:
yeah, but maybe this improvement should be part of another PR?

Reply:
Agree this will change the behavior and I don't want the migration blocked by an issue not "strictly" related to it.